merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -141,8 +155,8 @@ public Map<String, SerializedValue> getActivityEffectiveArguments(final String m
          InvalidArgumentsException
   {
     try {
-      return this.loadConfiguredMissionModel(missionModelId)
-                 .getActivityEffectiveArguments(activity.getTypeName(), activity.getArguments());
+      return this.loadUnconfiguredMissionModel(missionModelId)
+                 .getActivityEffectiveArguments(activity);
     } catch (final MissionModelFacade.NoSuchActivityTypeException ex) {
```

Comment:
I noticed that we have two versions of `NoSuchActivityTypeException`s, one from `MissionModelFacade` (seen here), and one in `MissionModelService` (seen in the `validateActivityInstantions` above). The behavior in both seem to be approximately the same, and neither depend on anything from the class/interface they reside in. Not this PR, but we should talk about potentially pulling these out to some kind of top-level exception class.

Reply:
After inlining, there is only one NoSuchActivityType exception - I've left it as `MissionModelService` because I see it as part of the interface. (I wouldn't put up a fight about pulling it out into a separate class - but I feel that there's no longer a compelling reason to, since the "duplicate" exception has now been deleted).